# Service to save a subscribed point cloud as a PCD file
# Use this service when taking empty container point clouds
string container #bin_a, bin_b, bin_c, tote, amnesty
---
# Return file path
string file_path